### DISCUSSION TITLE
[Feature] iOS App - Tap on Time to move to top

### DISCUSSION THREAD

**Discussion by vcdwelt** [immich-app/immich#9548] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Maybe it is possible to add the feature in iOS App, if you tap on the time, the App scrolls to top. This is in most iOS Apps the default behaviour.

### Platform

- [ ] Server
- [ ] Web
- [x] Mobile

**Comment by bo0tzz** [comment]
#1590
